guvnor-webapp/pom.xml

```diff
@@ -191,22 +192,6 @@
     <!-- Guvnor -->
     <dependency>
       <groupId>org.guvnor</groupId>
-      <artifactId>guvnor-asset-mgmt-api</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.guvnor</groupId>
-      <artifactId>guvnor-asset-mgmt-backend</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.guvnor</groupId>
-      <artifactId>guvnor-asset-mgmt-client</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.guvnor</groupId>
```

Comment:
These `asset-mgmt` deps were duplicated (they are declared again later in the pom).